compiler/testSuite/12-narrow/while02-v.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/io;
+
+public function main() {
+   int|boolean count = 3;
+   while count is int {
+      int i = count;
+      io:println(i); // @output 3
+                     // @output 2
+                     // @output 1
+      if count > 1 {
+         count = count - 1;
+      }
+      else {
+         count = false;
+      }
+   }
+   // boolean _ = count;
```

Comment:
@jclark Shouldn't `count` be narrowed to `false` here ?

Reply:
I don't think the spec says that yet.

Please add a spec issue with this example and subject "Normal completion of while-stmt should be able to cause variable type narrowing".

If you don't have a `break`, then after the `while` completes normally the loop condition while be false.

Reply:
crated https://github.com/ballerina-platform/ballerina-spec/issues/1019